src/services/inlayHints.ts

```diff
@@ -423,4 +430,13 @@ export function provideInlayHints(context: InlayHintsContext): InlayHint[] {
         }
         return true;
     }
+
+    function getNodeDisplayPart(text: string, node: Node): InlayHintDisplayPart {
+        const sourceFile = node.getSourceFile();

```

Comment:
A little unfortunate that you need to grab the source file for each parameter. Any way you can grab the source file of the current signature once?